src/db_options.rs

```diff
@@ -1208,6 +1216,28 @@ impl Default for Options {
     }
 }
 
+impl FlushOptions {
+    pub fn new() -> FlushOptions {
+        FlushOptions::default()
+    }
+
+    pub fn set_wait(&mut self, wait: bool) {
```

Comment:
And here as well please. Also, It could be cool to add a doc-test here.

Reply:
Done.